monai/bundle/utils.py

```diff
@@ -114,6 +114,7 @@
         "run_name": None,
         # may fill it at runtime
         "execute_config": None,
+        "dataset_log": True,
```

Comment:
I personally feel no need to add this arg in the default MLFlow settings.
What do you think?

Thanks.

Reply:
Deleted these part from the `utils` file, since the mlflow data tracking is an experiment API and may change in the future.